data/dapps/compound-finance.json

```diff
@@ -1,5 +1,11 @@
 {
-  "alias": "compound-finance",
-  "name": "Compound",
```

Comment:
You still need a dApp name if you will have the user first select the dApp and then the subselection for example on the Market frontend.

Reply:
My thinking was that the market UI will remain the same and the OEV Rewards select field will contain all of the relevant `title` options. I expect it will take some time until this becomes too messy and I'd defer subselection to a later stage.